README.md

```diff
@@ -62,6 +62,10 @@ If your application is running outside of Google Cloud Platform, such as locally
 
 3. Alternatively, if you are running your application on a machine where your are using the [`gcloud` command line tools][gcloud-sdk], and are logged using `gcloud auth login`, you already have sufficient credentials, and a service account key is not required.
 
+4. Generate a `source-context.json` file to share repository information with the cloud debugger service.
+
+        gcloud preview app gen-repo-info-file
+
```

Comment:
It would be good to add more information about what this file is.
> The gcloud preview app gen-repo-info-file command generates a file named source-context.json, which contains information about the version of the source code used to build the application. When you open the Stackdriver Debugger in the Cloud Platform Console, it uses the information in this file to display the correct version of the source

This is copied from https://cloud.google.com/debugger/docs/setting-up-go-on-compute-engine.